setup.py

```diff
@@ -1,28 +1,25 @@
 import os
 from distutils.core import setup
-
+from watson import __version__
```

Comment:
I'm totally fine with specifying `__version__` and `__author__` in `__init__.py`, and importing that directly in `setup.py`. I've taken this approach in plenty of other open-source projects, with no problems.

The only think to be aware of is that `__init__.py` cannot fail with an `ImportError` if a dependency (such as Django) isn't installed. Given that `__init__.py` is completely empty, it's safe to import.

@amureki - Keep things simple - just import the `__version__` and `__author__` from `__init__.py`. :)